tvac/client.py

```diff
@@ -0,0 +1,66 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
+
+class EnsemblError(Exception):
+    def __init__(self, status_code, error_text):            
```

Comment:
In production, providing a default URL that corresponded to a live server would probably be dangerous and/or confusing.